spec/cypress/app_commands/scenarios/no_mentors_for_participants.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+school = School.find_by(name: "Hogwarts Academy")
+if school.present?
+  school.mentor_profiles.each do |profile|
+    EarlyCareerTeacherProfile.where(mentor_profile: profile).update_all(mentor_profile_id: nil)
+    profile.user.destroy!
+  end
+  school.reload
+end
```

Comment:
What happens if the school is not present?

Surely we can make sure it exists, this is our test setup after all.

Reply:
Oh, I see, we are relying on another file being run before this one. A bit scary, but not too bad I guess.

I would still hope that the school was always present - otherwise the test cannot possibly work, right?

Reply:
Yes, just wanted a test to prove this was doing the right thing

Reply:
Comment added to file